addons.yaml

```
@@ -61,6 +61,13 @@ microk8s-addons:
       supported_architectures:
         - amd64
 
+    - name: "nfs"
+      description: "NFS Server Provisioner"
+      version: "1.4.0"
+      check_status: "statefulset.apps/nfs-server-provisioner"
```

```suggestion
      check_status: "statefulset.apps/nfs-server-provisioner"
      confinement: "classic"
```
Let's add confinement type since this will be a classic only addon for now.